macros/models/_sector/cex/cex_evms.sql

```diff
@@ -0,0 +1,18 @@
+{% macro cex_evms(cex_addresses, blockchain, traces) %}
+
+SELECT '{{blockchain}}' AS blockchain
+, address
+, cex_name
+, distinct_name
+, added_by
+, added_date
+, MIN(t.block_time) AS first_used
+FROM {{traces}} t
+INNER JOIN {{addresses}} a ON a.address = t.to
+LEFT JOIN {{this}} b ON a.address = b.address
+    AND b.address IS NULL
```

Comment:
`{{this}}` usage should be within `{% if is_incremental() %}` blocks, so it only looks for target if it has been built in prior full historical runs. this is especially true in CI runs, as we start fresh each run using the unique git commit hash as a prefix